types/svg-render/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for svg-render 1.2.0
+// Project: https://github.com/catdad-experiments/svg-render
+// Definitions by: Sergey Chernov <https://github.com/sergeychernov/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+ export interface SvgRenderOptions {
+    buffer: Buffer;
+    width?: number;
+    height?: number;
+    expandUseTags?: boolean;
+}
+export default function svgRender(options: SvgRenderOptions): Buffer;
```

Comment:
This is why the lint rule is complaining; it's not an ES module, it's a CJS module.

```suggestion
namespace svgRender {
  export interface SvgRenderOptions {
      buffer: Buffer;
      width?: number;
      height?: number;
      expandUseTags?: boolean;
  }
}

function svgRender(options: SvgRenderOptions): Buffer;

export = svgRender;
```